android/src/main/java/com/reactnative/googlecast/GoogleCastModule.java

```diff
@@ -258,6 +258,21 @@ private void setupCastListener() {
     mSessionManagerListener = new GoogleCastSessionManagerListener(this);
   }
 
+  @ReactMethod
+  public void getCurrentDevice(final Promise promise) {
+    getReactApplicationContext().runOnUiQueueThread(new Runnable() {
+      @Override
+      public void run() {
+        WritableMap map = Arguments.createMap();
+        map.putString("id:", mCastSession.getCastDevice().getDeviceId());
```

Comment:
the key should be without the `:`

Reply:
I'm going to work in IO's circumstances..